src/Bullet.js

```diff
@@ -64,7 +71,20 @@ class Bullet extends Phaser.GameObjects.Sprite {
         ) {
           this.kill();
         }
-      } else if (!this.data.bulletManager.bulletBounds.intersects(this)) {
+      } else if (
+        !Phaser.Geom.Intersects.RectangleToRectangle(
+          this.data.bulletManager.bulletBounds,
+          // this is janky af. I don't recall the actual method to pull a bounds
+          // rect from a v3 arcade body. getBounds() didn't work so I brute
+          // forced it.
```

Comment:
Seems like body.getBounds accepts ArcadeBodyBounds object (Rectangle?) and sets resizes that to be the size of the bounds.
https://photonstorm.github.io/phaser3-docs/physics_arcade_Body.js.html#line-1193

Reply:
Got it, ArcadeBodyBounds is compatible with a Rectangle
```javascript
/**
 * @typedef {object} ArcadeBodyBounds
 *
 * @property {number} x - [description]
 * @property {number} y - [description]
 * @property {number} right - [description]
 * @property {number} bottom - [description]
 */
```

Reply:
So it would do basically the same as you do here, but with an existing rectangle.
We could probably create a rectangle in the constructor and reuse it, to avoid creating a new one.